### PR TITLE
Use gene meta data in from referenceGenes

### PR DESCRIPTION
**Pull request by kalletlak**
Fix https://github.com/cBioPortal/cbioportal/issues/6799

### CODE REVIEW

File: src/pages/resultsView/coExpression/CoExpressionTab.tsx [hunk 1]
```diff
@@ -192,23 +192,50 @@ export default class CoExpressionTab extends React.Component<ICoExpressionTabPro
...
+
+                    return Promise.resolve(_.reduce(data, (acc, datum) => {
+                        let geneticEntityName: string | undefined = undefined;
+                        if (geneset !== undefined) {
+                            geneticEntityName = geneset.name;
+                        } else if (entrezGeneIdToReferenceGene[datum.geneticEntityId] !== undefined) {
+                            geneticEntityName = entrezGeneIdToReferenceGene[datum.geneticEntityId].hugoGeneSymbol;
+                        }
+
+                        if (geneticEntityName !== undefined) {
```
Comment (alisman) [inline src/pages/resultsView/coExpression/CoExpressionTab.tsx]: out of curiosity, why do have check for existence of name in order to get cytoband?

Comment (kalletlak) [reply src/pages/resultsView/coExpression/CoExpressionTab.tsx]: not required. updated

**Comment by alisman** [comment]
@pvannierop could you review this PR please with an eye to genesets?   The problem we are fixing is that it is very expensive to supply coexpression datapoints with the gene name/cytoband on server side.   Turned out to be a big bottleneck.

**Comment by kalletlak** [comment]
Created and merged another duplicate pr (https://github.com/cBioPortal/cbioportal-frontend/pull/2871) to resolve circleci api cache issue. Closing this one.

**Pull request closed without merging**